style.css

```diff
@@ -721,3 +720,60 @@
   padding-left: 10px;
   margin-top: 20px;
 }
+
+.mobile {
+  background: #3c3a39;
+  color: #fff5e1;
+  font-size: 25px;
+  font-family: "Inter", sans-serif;
```

Comment:
- [ ] You did an amazing job adding mobile menu and styling it but please stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). Keep coding 💪 

Here you have a screenshot of your hamburger menu:

![image](https://user-images.githubusercontent.com/74506933/157294893-cbecb72e-a72a-4577-8e1f-5b52ad757f00.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/157294949-6d2f1885-75a1-4cd8-84bc-71d1c31719b0.png)